src/GZip.jl

```diff
@@ -414,7 +416,7 @@ function readall(s::GZipStream, bufsize::Int)
             if length(buf) > len
                 resize!(buf, len)
             end
-            return bytestring(buf)
+            return UTF8String(copy(buf))
```

Comment:
I think this is changing the return type? Is there a `Compat.bytestring` that maintains the 0.4 return type when run on that version?